packages/components/psammead-navigation/src/DropdownNavigation/__snapshots__/index.test.jsx.snap

```diff
@@ -980,54 +1037,79 @@ exports[`Dropdown navigation should render correctly when open 1`] = `
         role="listitem"
       >
         <a
+          aria-labelledby="dropdownNavigation-Video"
           class="emotion-6 emotion-7"
           href="/pidgin/media/video"
         >
-          Video
+          <span
+            id="dropdownNavigation-Video"
+          >
+            Video
+          </span>
         </a>
       </li>
       <li
         class="emotion-4 emotion-5"
         role="listitem"
       >
         <a
+          aria-labelledby="dropdownNavigation-Audio"
           class="emotion-6 emotion-7"
           href="/pidgin/media/audio"
         >
-          Audio
+          <span
+            id="dropdownNavigation-Audio"
+          >
+            Audio
+          </span>
         </a>
       </li>
       <li
         class="emotion-4 emotion-5"
         role="listitem"
       >
         <a
+          aria-labelledby="dropdownNavigation-Sport"
           class="emotion-6 emotion-7"
           href="/pidgin/sport"
         >
-          Sport
+          <span
+            id="dropdownNavigation-Sport"
+          >
+            Sport
+          </span>
         </a>
       </li>
       <li
         class="emotion-4 emotion-5"
         role="listitem"
       >
         <a
+          aria-labelledby="dropdownNavigation-Entertainment"
           class="emotion-6 emotion-7"
           href="/pidgin/topics/1c3b60a9-14eb-484b-a750-9f5b1aeaac31"
         >
-          Entertainment
+          <span
+            id="dropdownNavigation-Entertainment"
+          >
+            Entertainment
+          </span>
         </a>
       </li>
       <li
         class="emotion-4 emotion-5"
         role="listitem"
       >
         <a
+          aria-labelledby="dropdownNavigation-Most popular"
```

Comment:
Id text looks like it's missing a dash between Most and popular?

Reply:
Not that this is needed here, only needed when we have current page

Reply:
Now code is smart enough to replace spaces with dashes 😄